packages/@okta/vuepress-site/docs/concepts/policies/index.md

```diff
@@ -4,116 +4,151 @@ meta:
 
-* [IdP Discovery Policy](/docs/reference/api/policy/#idp-discovery-policy): Determines where to route users when they attempt to sign in to your org. You can route users to a variety of [identity providers](/docs/guides/add-an-external-idp/).
+The [`system` attribute](/docs/reference/api/policy/#policy-object) determines whether a system or a user created the policy. Default policies and default rules are the only policies and rules that have this attribute. The `system` attribute set to `TRUE` on default policies or rules indicates that those policies and rules are system-created policies, and you can't delete them.
 
-  > **Note:** This policy isn't for performing authentication or authorization. It's used only to determine where a user is routed. You can't control access with an IdP Discovery Policy.
+## Policy types
 
-## Policy use cases
+You can configure policies for sign-on, passwords, enrollment, and API access. You can also use IdP discovery policies to create routing rules.
 
-**Control who can access your app**
+### Sign-on policies
 
-The global session policy controls the manner in which a user is allowed to sign in to Okta. It identifies the user and specifies the length of their session. You can [configure a global session policy](/docs/guides/configure-signon-policy/main/) to require any of the [factors that you set up](https://help.okta.com/okta_help.htm?type=oie&id=csh-configure-authenticators). Then use the primary and secondary factor conditions in a rule to define which factors are evaluated. For example, add a rule that prompts for additional factors when you want only users who are inside your [corporate network](/docs/reference/api/policy/#network-condition-object) to have access.
+[Sign-on policies and rules enforce policies and rules so users sign in with the right level of access that you provide. [Global session policies](#global-session-policies) and [authentication policies](#authentication-policies) together determine the level of confidence that the user signing in is also the person who owns the account.
 
-**Control how users access your app**
+#### Global session policies
 
-In addition to the global session policy, there is an authentication policy for each application that determines the extra levels of authentication that you may want performed before a user can access an application. [Add a rule](/docs/guides/configure-access-policy), for example, to prompt groups that are assigned to your app to re-authenticate after 60 minutes.
+A [global session policy](/docs/reference/api/policy/#global-session-policy) controls who has access and how a user gains access to Okta. See [Global session policies](https://help.okta.com/oie/en-us/content/topics/identity-engine/policies/about-okta-sign-on-policies.htm) and [Configure a global session policy and an authentication policy](/docs/guides/configure-signon-policy/main/).
```

Comment:
(https://help.okta.com/oie/en-us/content/topics/identity-engine/policies/about-okta-sign-on-policies.htm)  - should be an alias

```suggestion
A [global session policy](/docs/reference/api/policy/#global-session-policy) controls who has access and how a user gains access to Okta. See [Global session policies](https://help.okta.com/oie/en-us/content/topics/identity-engine/policies/about-okta-sign-on-policies.htm) and [Configure a global session policy and an authentication policy](/docs/guides/configure-signon-policy/main/).
```